transfer.md

```diff
@@ -83,18 +89,39 @@ Those Docker commands currently do not have equivalents in `podman`:
 | `docker events`   ||
 | `docker network`  ||
 | `docker node`     ||
-| `docker plugin`   |podman does not support plugins.  We recommend you use alternative OCI Runtimes or OCI Runtime Hooks to alter behavior of podman.|
+| `docker plugin`   | podman does not support plugins.  We recommend you use alternative OCI Runtimes or OCI Runtime Hooks to alter behavior of podman.|
 | `docker rename`   | podman does not support rename, you need to use `podman rm` and  `podman create` to rename a container.|
 | `docker secret`   ||
 | `docker service`  ||
 | `docker stack`    ||
 | `docker swarm`    | podman does not support swarm.  We support Kubernetes for orchestration using [CRI-O](https://github.com/kubernetes-sigs/cri-o).|
 | `docker system`   ||
-| `docker volume`   | podman does not support volumes.  Volumes should be built on the host operating system and then volume mounted into the containers.|
+| `docker volume`   | podman currently supports file volumes.  Future enhancement planned to support Docker Volumes Plugins
 
 ## Missing commands in Docker
 
 The following podman commands do not have a Docker equivalent:
 
+* [`podman generate`](./docs/podman-generate.1.md)
+* [`podman generate kube`](./docs/podman-generate-kube.1.md)
+* [`podman image exists`](./docs/podman-image-exists.1.md)
+* [`podman image sign`](./docs/podman-image-sign.1.md)
+* [`podman image trust`](./docs/podman-image-trust.1.md)
 * [`podman mount`](./docs/podman-mount.1.md)
+* [`podman play`](./docs/podman-play.1.md)
+* [`podman play kube`](./docs/podman-play-kube.1.md)
+* [`podman pod`](./docs/podman-pod.1.md)
+* [`podman pod create`](./docs/podman-pod-create.1.md)
+* [`podman pod exists`](./docs/podman-pod-exists.1.md)
+* [`podman pod inspect`](./docs/podman-pod-inspect.1.md)
+* [`podman pod kill`](./docs/podman-pod-kill.1.md)
+* [`podman pod pause`](./docs/podman-pod-pause.1.md)
+* [`podman pod ps`](./docs/podman-pod-ps.1.md)
+* [`podman pod restart`](./docs/podman-pod-restart.1.md)
+* [`podman pod rm`](./docs/podman-pod-rm.1.md)
+* [`podman pod start`](./docs/podman-pod-start.1.md)
+* [`podman pod stop`](./docs/podman-pod-stop.1.md)
+* [`podman pod top`](./docs/podman-pod-top.1.md)
+* [`podman pod unpause`](./docs/podman-pod-unpause.1.md)
+* [`podman varlink`](./docs/podman-varlink.1.md)
```

Comment:
Do we want to document this one here? It's only really meant to be invoked from systemd

Reply:
I think we should document all commands that are different then docker.